lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -98,6 +98,24 @@ def operating_systems
       @category_types
     end
 
+    # aws does not provide any json-sourced network_capacity, so for now we provide this hardcoded lookup
+    # see https://aws.amazon.com/ec2/instance-types/, under 'Instance Types Matrix', under 'Networking Performance'
+    def self.get_network_capacity(api_name)
+      throughput = @Network_Performance[api_name]
+      if not throughput
+        logger.warn "Unknown network throughput for instance type #{api_name}"
+      end
+      throughput
+    end
+    def self.get_network_mbps(throughput)
+      if throughput
+        network_mbps = @Network_Throughput_Bytes_Per_Hour[throughput]
+      else
+        logger.warn "Unknown network throughput #{throughput}"
```

Comment:
nit: if we were following the same pattern, this logger.warn will be on '!network_mbps', not on 'throughput'?